src/main/java/org/owasp/html/ExtensibleHtmlStreamRenderer.java

```diff
@@ -315,7 +324,7 @@ public final void text(String text) {
         }
     }
 
-    private final void writeText(String text) throws IOException {
+    protected final void writeText(String text) throws IOException {
```

Comment:
if you make a final then it can not be overridden

Reply:
done